build-system/tasks/visual-diff/index.js

```diff
@@ -131,10 +135,20 @@ async function launchPercyAgent() {
   }
 
   const env = argv.percy_agent_debug ? {LOG_LEVEL: 'debug'} : {};
-  percyAgentProcess_ = execScriptAsync('npx percy start', {
-    cwd: __dirname,
-    env: Object.assign(env, process.env),
-    stdio: ['ignore', process.stdout, process.stderr],
+  percyAgentProcess_ = execScriptAsync(
```

Comment:
Sanity check: You're calling `execScriptAsync` because the agent process needs to be a daemon process, correct? Is there a way to call it normally and have it sit in the background?

Reply:
I believe this is the way to do what you're asking (behind the scenes it's `childProcess.spawn`), as opposed to `childProcess.spawnSync` which leaves the program hanging until the child process exits

Reply:
Hmm. I'm confused by https://www.npmjs.com/package/@percy/agent#commands because it doesn't list `percy start` as a command.  No worries, since this works as written.

Reply:
Correct! I mentioned it to them in an issue - it's just a missing piece of documentation